<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration_ui</class>
 <widget class="QWidget" name="Registration_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像配准</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</normaloff>:/QtWidgetsApplication3/bin/icon/InSARapp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>相关配准</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>工程选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>数据节点选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>主图像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_masterIndex"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileedit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>插值倍数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_interptimes"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>子块大小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_blocksize"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,5">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>DEM辅助配准</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>工程选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_project"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>数据节点选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_node"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>主图像</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_masterIndex2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dstNode"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="5,5">
         <item>
          <widget class="QProgressBar" name="progressBar_2">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox_2">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
